docs/docs/t-digest.md

```diff
@@ -62,62 +62,56 @@ You measure the IP packets transferred over your network each second and try to
 
 ## Examples
 
-#### Creating a t-digest
+We'll demonstrate creating a t-digest with an initial compression of 100, adding items to it, and checking the estimated percentile of a value. Note that the `COMPRESSION` argument is used to specify the tradeoff between accuracy and memory consumption. The default is 100. Higher values mean more accuracy. Note also that unlike some of the other probabilistic data structures, the `TDIGEST.ADD` command will not create a new structure if the key does not exist.
 
-```
-> TDIGEST.CREATE my-tdigest COMPRESSION 100
-```
+{{< clients-example tdigest_tutorial tdig_start >}}
+> TDIGEST.CREATE bikes:sales COMPRESSION 100
+OK
+> TDIGEST.ADD bikes:sales 21
+OK
+> TDIGEST.ADD bikes:sales 150 95 75 34
+OK
+{{< /clients-example >}}
 
-The `COMPRESSION` argument is used to specify the tradeoff between accuracy and memory consumption. The default is 100. Higher values mean more accuracy.
-
-#### Adding a single element to the t-digest:
-```
-> TDIGEST.ADD my-tdigest 20.9
-```
-
-#### Adding multiple elements to the t-digest:
-```
-> TDIGEST.ADD my-tdigest 308 315.9
-```
 
 You can repeat calling [TDIGEST.ADD](https://redis.io/commands/tdigest.add/) whenever new observations are available
 
 #### Estimating fractions or ranks by values
 
 Another helpful feature in t-digest is CDF (definition of rank) which gives us the fraction of observations smaller or equal to a certain value. This command is very useful to answer questions like "*What's the percentage of observations with a value lower or equal to X*".
 
->More precisely, `TDIGEST.CDF` will return the estimated fraction of observations in the sketch that are smaller than X plus half the number of observations that are equal to X
-
-Let's illustrate this with an example: if we have a set of observations of people's age with gaussian distribution, we can ask a question like "What's the percentage of people younger than 50 years?"
+>More precisely, `TDIGEST.CDF` will return the estimated fraction of observations in the sketch that are smaller than X plus half the number of observations that are equal to X. We can also use the `TDIGEST.RANK` command, which is very similar. Instead of returning a fraction, it returns the **number** of observations in the sketch that are smaller than X plus half the number of observations that are equal to X, or in other words - the estimated rank of a value. The `TDIGEST.RANK` command is also variadic.
```

Comment:
```suggestion
>More precisely, `TDIGEST.CDF` will return the estimated fraction of observations in the sketch that are smaller than X, plus half the number of observations that are equal to X. We can also use the `TDIGEST.RANK` command, which is very similar. Instead of returning a fraction, it returns the *number* of observations in the sketch that are smaller than X, plus half the number of observations that are equal to X, or, in other words, the estimated rank of a value. The `TDIGEST.RANK` command is also variadic.
```

Reply:
Please add "estimated":
> Instead of returning a fraction, it returns the ----estimated---- **number** of observations.

> The `TDIGEST.RANK` command is also variadic.
Consider using a simpler explanation. I'm not sure every reader knows what "variadic" means here.